### DISCUSSION TITLE
[Suggestion] Add option to change lower angle tilt limit

### DISCUSSION THREAD

**Discussion by CodeAndGin** [runelite/runelite#15791] [Ideas]
So the camera can only tilt so low, and it has forced near vertical sections near cliffs and mountains (like on route to death plateau for instance). An option to remove the forced sections and change the minimum angle (say to the angle you can tilt to in RS3) would be appreciated to avoid weird forced camera movement, and be able to play zoomed in a bit nicer. I understand that a lot of objects do not have undersides, so would render weird, but this doesn't bother me too much (and is a problem in RS3, look at the rooves in new lumbridge at the minimum tilt for example) and mostly this would be fine for visibility, as the inside of cliffs wont block vision

**Comment by Enriath** [comment]
This is really two separate requests.

Horizontal camera has been rejected before, but not for a long time (see #6979). I'm not entirely sure if our stance on it has changed, but that's how it is at the moment.

Disabling the camera clipping protection stuff would probably be fine from a rules perspective, but I can't help but imagine it would cause issues with clicking on things based on what I remember about how the game decides what to click on (though it has been a while). Either way, it would probably be an internal change to make it happen.

**Comment by CodeAndGin** [comment]
That linked reddit post has a solution for the afk agility strat that doesn't involve a horizontal camera. I literally only want it so that if I zoom in close I don't have a 6x6 tile clickspace lol, Understandable if there are other weird things that make this disallowed tho.

I remember being able to clip the camera through the cliffs near the rag and bone man, so after I'm done my slayer task ill swing round there and see if there's any weird click collisions, though only having one testing spot probably wont give much usable data

**Comment by CodeAndGin** [comment]
Alright, tested near rag and bone man - clicks seem to work fine if through the underside of a mesh - but if there's any of the mesh your clipping through, you can click on that. Seems the collision is just on the visible side

**Comment by CodeAndGin** [comment]
To replicate - on the road from varrock to rag and bone man,  where the road does the weird meander around a patch of grass, if you stand on the grass and zoom out, you can clip through some of the terrain

**Comment by PortalInTime** [comment]
I would absolutely _love_ to have this addition to the game. I hope it gets considered.

**Comment by Advistane** [comment]
Bumping this based on recent discussion in development discord.
![aqmfct](https://user-images.githubusercontent.com/35465504/205395511-737db961-60ad-43f6-8000-0a8e6ddcf276.png)
